packages/kbn-rule-data-utils/src/alerts_as_data_rbac.ts

```diff
@@ -23,6 +23,7 @@ export const AlertConsumers = {
   SLO: 'slo',
   SIEM: 'siem',
   UPTIME: 'uptime',
+  ML: 'ml',
```

Comment:
Is this alert consumer related to this Kibana privilege?
![image](https://github.com/elastic/kibana/assets/12370520/c1204d55-baf5-453c-80b4-14d3f30ad490)

Reply:
I assume it is not. @XavierM could you please confirm?

Reply:
@XavierM what feature privileges are needed to see the 'View rule details' action? In testing, I am only seeing it when the case actions are visible:

With 'All' on Stack Rules, 'Read' on Rule Settings and 'Read' on Stack Cases I don't see the 'View rule details'

<img width="214" alt="image" src="https://github.com/elastic/kibana/assets/7405507/b063ee2b-93c7-4a2b-9de7-025fc58b1eab">

But if I switch Stack Cases to 'All' I then see all three actions:

<img width="231" alt="image" src="https://github.com/elastic/kibana/assets/7405507/33a76be5-d014-4b65-914c-f867de888450">

Reply:
@peteharverson That's sound like a bug, let me check the code

Reply:
@darnautov, it is  because we will use AlertConsumer to set our featureIds. it will be better to keep it similar than the ml feature id

Reply:
Consumer `ml` looks good. Same as the feature id of the plugin

Reply:
@darnautov was a code change required here? I no longer see this issue, but I wonder if it's a side effect of hiding the cases actions?

Reply:
There was an incorrect condition before that was fixed in 0a5fcde63daf26730b823975ad135ab6af374bbf